src/data-scraper/index.ts

```diff
@@ -29,19 +40,19 @@ function isSuccessStatusCode(statusCode: number | undefined): boolean {
 export async function validateConnectivity(): Promise<void> {
   const url = getSysdigUrl();
   const header = getSysdigAuthHeader();
+  const clusterName = config.SYSDIG_CLUSTER_NAME;
+
   const reqOptions: NeedleOptions = {
     agent: httpsAgent,
     headers: {
       authorization: header,
     },
     timeout: 10_000,
   };
-
   const limit: number = 1;
-  const cursor: string = '';
   const { response } = await retryRequest(
     'get',
-    `${url}?limit=${limit}&cursor=${cursor}`,
+    `${url}?clusterName=${clusterName}&limit=${limit}`,
```

Comment:
this url was `${url}?limit=${limit}&cursor=${cursor}` for legacy so will not work here?

Reply:
oh I see it below!